### PR TITLE
Feature/card homepage

### PR DESCRIPTION
**Pull request by alipoorseyed**

### CODE REVIEW

File: src/components/Badge.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      if (counter === 3 && i !== 0) {
+        result = ',' + result;
+        counter = 0;
+      }
+    }
+
+    setFormattedPrice(result);
+  }, [price]);
+
+  return <div className="bg-[#831747] text-[#F8D4E4] text-[11px] rounded-xl px-2 py-[2px] font-[400]">{formattedPrice + " تومان"}</div>;
```
Comment (ArshiaRzi) [inline src/components/Badge.tsx]: color pallet must be like the colors in tailwind.config.js

File: src/components/Card_Homepage.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  imgsrce : string;
+}
+
+export const Card_Homepage: React.FC<CardHomepageProps> = ({ title , imgsrce = ""}) => {
+  return (
+    <div className="flex flex-col gap-3 justify-start items-stretch w-[300px] relative">
+      <Like_Icon/>
+      <img className='rounded-xl' src={imgsrce} alt="" />
+      <div className="flex flex-row justify-between items-center">
+        <p className='text-[14px] font-[400] text-[black]'>{title}</p>
```
Comment (ArshiaRzi) [inline src/components/Card_Homepage.tsx]: use rem instead of px

File: package.json [hunk 3]
```diff
@@ -28,6 +28,6 @@
     "tailwindcss": "^3.4.11",
     "typescript": "^5.5.3",
     "typescript-eslint": "^8.0.1",
-    "vite": "^5.4.1"
+    "vite": "^5.4.5"
```
Comment (ArshiaRzi) [inline package.json]: don't change the version set is as it was
5.4.1

File: src/components/Badge.tsx (hunk 1 above)
Comment (alipoorseyed) [reply src/components/Badge.tsx]: ok baby 👍

File: src/components/Card_Homepage.tsx (hunk 2 above)
Comment (alipoorseyed) [reply src/components/Card_Homepage.tsx]: ok i will fix it .

**Comment by alipoorseyed** [comment]
۳

File: src/components/Badge.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      if (counter === 3 && i !== 0) {
+        result = ',' + result;
+        counter = 0;
+      }
+    }
+
+    setFormattedPrice(result);
+  }, [price]);
+
+  return <div className="bg-danger-600 text-danger-100 text-[11px] rounded-xl px-2 py-0.5 font-normal">{formattedPrice + " تومان"}</div>;
```
Comment (ArshiaRzi) [inline src/components/Badge.tsx]: text-[11px]???????

Comment (alipoorseyed) [reply src/components/Badge.tsx]: i use text-xs and fix it .

**Pull request merged**